src/applications/vaos/newAppointmentFlow.js

```diff
@@ -190,9 +183,8 @@ export default {
             dispatch(startDirectScheduleFlow(appointments));
             return 'clinicChoice';
           }
-        } else {
-          dispatch(startDirectScheduleFlow(appointments));
-          return 'clinicChoice';
+        } catch (error) {
+          captureError(error);
```

Comment:
If we can't get appointment history, should we `dispatch(startRequestAppointmentFlow())` in the catch block to send them to the request flow?

Reply:
Yeah, if you expand the file below this change you can see that if they fall through here we'll check that they're eligible for requests and start that flow.